genesis/data.go

```diff
@@ -0,0 +1,45 @@
+package genesis
+
+import "math/big"
+
+// DelegationData specify the delegation address and the balance provided
+type DelegationData struct {
+	Address string `json:"address,omitempty"`
+	Balance string `json:"balance,omitempty"`
+	address []byte
+	balance *big.Int
+}
+
+// InitialBalance provides information about one entry in the genesis file
+type InitialBalance struct {
+	Address        string          `json:"address"`
+	Supply         string          `json:"supply"`
+	Balance        string          `json:"balance"`
+	StakingBalance string          `json:"stakingbalance"`
+	Delegation     *DelegationData `json:"delegation"`
+
+	address        []byte
```

Comment:
These fields are just so we would not use separate structures for decoding the json file? Wouldn't a custom unmarshal function be cleaner?

Reply:
Partially, address can not be automatically converted by the json marshaler as we need the PubkeyConverter. Will change the *big.Int fields

Reply:
done for big.Int, address remains the same